aospy/data_loader.py

```diff
@@ -89,11 +88,7 @@ def set_grid_attrs_as_coords(ds, set_time_vars=True):
     Dataset
         Dataset with grid attributes set as coordinates
     """
-    if set_time_vars:
-        int_names = GRID_ATTRS.keys()
-    else:
-        int_names = (set(GRID_ATTRS.keys()) -
-                     set(TIME_VAR_STRS))
+    int_names = GRID_ATTRS.keys()
     grid_attrs_in_ds = set(int_names).intersection(set(ds.coords) |
                                                    set(ds.data_vars))
     ds.set_coords(grid_attrs_in_ds, inplace=True)
```

Comment:
Would you mind taking care of #244 while you're at it by removing this use of `inplace`?

Reply:
Sure thing, that should be straightforward.